Makefile

```diff
@@ -73,9 +74,21 @@ require-sync:
 generate-website-docs:
 	cd $(WEBSITE_SCRIPT_DIR); go run generate.go
 
+.PHONY: test-artifacthub-script
+test-artifacthub-script:
```

Comment:
can we just call this "make test" and have all tests run there? is there a reason for each of these to have their own make target?